index.html

```diff
@@ -50,6 +52,194 @@ <h4 class="connect">LET'S CONNECT</h4>
+          </ul>
+        </div>
+
+        <div>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+
+      </div>
+
+      <!-- Multi-post-stories -->
+      <div class="cards">
+        <div class="cards-img"><img src="./img/snapshoot-b.png" alt="card-image"></div>
+        <h2 class="h2">Multi-Post<br>Stories</h2>
+
+        <ul class="top">
+          <li class="canopy">CANOPY</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">Back End Dev</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+
+        <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="projects">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+
+      </div>
+
+      <!-- Tonic-B -->
+      <div class="cards">
+        <div class="cards-img"><img src="./img/snapshoot-c.png" alt="card-image"></div>
+        <h2 class="h2">Tonic</h2>
+
+        <ul class="top">
+          <li class="canopy">CANOPY</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">Back End Dev</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+
+        <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="projects">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+
+      </div>
+
+      <!--Multi-post-stories-B -->
+      <div class="cards">
+        <div class="cards-img"><img src="./img/snapshoot-d.png" alt="card-image"></div>
+        <h2 class="h2">Multi-Post<br>Stories</h2>
+
+        <ul class="top">
+          <li class="canopy">CANOPY</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">Back End Dev</li>
+          <li><img src="./img/counter.png" alt="ash-counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+
+        <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="projects">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="submit" class="btn">See Project</button>
+        </div>
+
+      </div>
+
+    </section>
+
+    <!-- About-Section -->
+    <section class="container-section">
+
+      <div class="intro2">
+        <h1>About<br>Myself</h1>
```

Comment:
Kindly change the `h1` tag to be any heading between `h2` to `h6` for example `h2`, since the `h1` should only exist once on your page for the main heading.